Unit.java

```diff
@@ -1,19 +1,22 @@
 public class Unit {
+    String name;
     final int dmg;
     int hp;
```

Comment:
En azert orulnek a private-nek.

Reply:
Igazabol ha van egy `final`-unk, ami immutable/primitiv dologra mutat, akkor rendben van ez, Tehat pl a name eseteben.  De pl a hp ele indenkeppen kellene a private.Valoszinuleg szebb lenne ha a dmg elott is lenne, a nev-nel meg izles kerdese, de ha nem private, akkor mondjuk nem art, ha final.